goodgame-persistence/rdb/src/main/kotlin/people/songpagu/goodgame/jpa/domain/matching/collection/DistrictCollection.kt

```diff
@@ -3,5 +3,10 @@ package people.songpagu.goodgame.jpa.domain.matching.collection
 import people.songpagu.goodgame.domain.matching.option.type.District
 
 data class DistrictCollection(
-    val values: List<District> = emptyList()
-)
+    val values: MutableList<District> = mutableListOf()
+) {
+    fun updateValues(districts: List<District>) {
+        this.values.clear();
+        this.values.addAll(districts)
+    }
```

Comment:
코틀린에서는 성능상에 문제가 없다면 아래와 같이 불변객체를 사용하는것을 더 추천하는편이얌

```suggestion
    var values: List<District> = listOf()
) {
    fun updateValues(districts: List<District>) {
        this.values = districts
    }
```